kubernetes/helm/values.prod.yaml

```diff
@@ -84,5 +84,6 @@ dug:
     image: koursaros/nboost
     image_tag: 0.3.9-pt
     api_port: 8000
+    model: nboost/pt-tinybert-msmarco
```

Comment:
The default model according to https://hub.docker.com/r/koursaros/nboost. Was either this method of being explicit on the model or adding logic to the template to handle the key not existing or having no value 🤷‍♂️